### PR TITLE
Merged "Xcode Build Configuration" and "Xcode Project".

### PR DESCRIPTION
**Pull request by peterthomashorn**
As suggested during initial publishing on Package Control and later on in a dedicated issue.

### CODE REVIEW

File: repository/x.json [hunk 1]
```diff
@@ -45,8 +45,9 @@
 			]
 		},
 		{
-			"name": "Xcode Build Configuration",
-			"details": "https://github.com/peterthomashorn/xcode-build-configuration-language-for-sublime-text",
+			"name": "Xcode Build Configuration and Project Syntax Definitions",
```
Comment (braver) [inline repository/x.json:48]: Lol, ok, bit long for a name but that works. "Xcode Builds and Projects" maybe?

**Comment by jrappen** [comment]
> as suggested ...

Do you expect people to look up your prior discussions? Would have been easier to just provide the links to those as does everybody else across GitHub.

File: repository/x.json (hunk 1 above)
Comment (peterthomashorn) [reply repository/x.json:48]: Yeah, I know, but somehow I would like to keep it clear this is only about syntax definitions, not full support of such or anything else. 😅

**Comment by peterthomashorn** [comment]
> > as suggested ...
> 
> Do you expect people to look up your prior discussions? Would have been easier to just provide the links to those as does everybody else across GitHub.

No. I had the tabs open for copying and pasting the links. I just simply forgot that because I was juggling with four repositories, some issues and pull requests simultaneously.

File: repository/x.json (hunk 1 above)
Comment (peterthomashorn) [reply repository/x.json:48]: In case an additional syntax is added I will definitely think about the title again to make it shorter.

**Pull request merged**